indicaAi/spec/models/local_rating_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe LocalRating, type: :model do
+  it 'Testing if value is integer - Should return true' do
+    note1 = LocalRating.new
```

Comment:
Aqui deve-se criar uma factory para o LocalRating e usála.

Reply:
devo criar uma factory a parte se for outra model?